src/main/resources/db/migration/0300/V349__FundingEntities.sql

```diff
@@ -0,0 +1,28 @@
+CREATE SCHEMA IF NOT EXISTS funder;
+
+CREATE TABLE funder.funding_entities
+(
+    id   BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL UNIQUE,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE TABLE funder.funding_entity_projects
+(
+    funding_entity_id BIGINT NOT NULL REFERENCES funder.funding_entities,
```

Comment:
Do we want ON DELETE CASCADE for funding entities too?

Reply:
Not quite following this one. What would be getting deleted that would cascade and delete a funding entity?

Reply:
Ohh sorry was thinking of the wrong table. Ya it does probably need to be added for both. Will update